web/plugin/feature/phonebook/import.php

```diff
@@ -154,8 +154,11 @@
 			if ($name && $mobile) {
 				// fixme anton - temporary - contacts not unique
 				// if ($c_pid = phonebook_number2id($uid, $mobile)) {
-				if (FALSE) {
+				$db_query = "SELECT id FROM " . _DB_PREF_ . "_featurePhonebook WHERE `mobile` = '$mobile' AND `uid` = $uid LIMIT 1";
```

Comment:
`$mobile` may not be in the same format with whats in the phonebook table, thats why I prefer to use `phonebook_number2id()`, or maybe just use LIKE, like this: https://github.com/antonraharja/playSMS/blob/master/web/plugin/feature/phonebook/fn.php#L104